src/tests/resistance/handshake.rs

```diff
@@ -25,6 +28,9 @@ use crate::{
 const WS_HTTP_HEADER_MAX_SIZE: usize = 7700;
 const WS_HTTP_HEADER_INVALID_SIZE: usize = WS_HTTP_HEADER_MAX_SIZE + 300;
 
+// Number of seconds between unix and ripple epoch.
+const RIPPLE_EPOCH_OFFSET: u64 = 946684800;
```

Comment:
I'll just mention, I used this same number when doing the validator test.
https://github.com/runziggurat/xrpl/blob/main/src/tests/conformance/post_handshake/validators.rs#L23
Maybe something to think about, how to have it exist in only one place.

Reply:
Let's move it some constant in one of future PR?